### PR TITLE
add old help texts before we do testing for better ones

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/y2partitioner/widgets/lvm_vg_description.rb [hunk 1]
```diff
@@ -19,11 +21,15 @@ def init
...
       end
 
+      HELP_FIELDS = [:device, :size, :pe_size].freeze
       # @macro seeAbstractWidget
       def help
-        # TODO: proofread it and test it on real user, if it need improvement
-        _("Textual description of LVM Volume Group data and configuration. "\
-          "<b>PE Size</b> is physical extend size.")
+        header = _(
+          "<p>his view shows detailed information about the\nselected volume group.</p>" \
```
Comment (mvidner) [inline src/lib/y2partitioner/widgets/lvm_vg_description.rb]: **T**his view

Comment (jreidinger) [inline src/lib/y2partitioner/widgets/lvm_vg_description.rb]: thanks for catch, copy and paste error

File: src/lib/y2partitioner/widgets/disk_table.rb [hunk 2]
```diff
@@ -69,22 +71,16 @@ def header
         ]
       end
 
+      HELP_FIELDS = [:device, :size, :format, :encrypted, :type, :fs_type,
+                     :label, :mount_point, :start_cyl, :end_cyl].freeze
```
Comment (mvidner) [inline src/lib/y2partitioner/widgets/disk_table.rb]: This is error prone, just like defining the table columns elsewhere. We should eventually come up with something that is less likely to break if you need to add/move/remove a table column.

Comment (jreidinger) [inline src/lib/y2partitioner/widgets/disk_table.rb]: as I write it is just stupid copy and paste of old storage approach with fields. I add also TODO there for this, but now I am a bit under time pressure and I hope we do some real UX testing of helps and write to widgets what is not clear for users.

**Pull request merged**